deploy/sre-authorization/03-osd-sre-admin.ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: osd-sre-admin
```

Comment:
if we are using this setup for Online and Pro as well, would it make sense to replace OSD with managed?